discovery/swarmkit/swarmkit.go

```diff
@@ -129,7 +129,8 @@ func (d *Discovery) Watch(stopCh <-chan struct{}) (<-chan discovery.Entries, <-c
 					ch <- newEntries
 				}
 				currentEntries = newEntries
-			case <-eventsChan:
+			case e := <-eventsChan:
+				log.Debugf("SwarmKit discovery: received node event with action: %s", e.Action)
```

Comment:
Is this going to be super spammy in big clusters?

Reply:
We only receive events with type `node` (this is ensured by the events filter above). So as long as we don't have a lot of membership changes, this case won't be triggered. Perhaps it's still worth removing since it works fine and is not an error case.